test/controllers/EventReportControllerSpec.scala

```diff
@@ -63,7 +64,7 @@ class EventReportControllerSpec extends AsyncWordSpec with Matchers with Mockito
   before {
     reset(mockAuthConnector, mockJSONPayloadSchemaValidator)
     when(mockAuthConnector.authorise[Option[String]](any(), any())(any(), any())) thenReturn Future.successful(Some("Ext-137d03b9-d807-4283-a254-fb6c30aceef1"))
-    when(mockJSONPayloadSchemaValidator.validateJsonPayload(any(), any())) thenReturn Right(true)
+    when(mockJSONPayloadSchemaValidator.validatePayload(any(), any(), any())).thenReturn(Failure(new Exception("Test")))
```

Comment:
would it be better to test this with expected exception class ? (EventReportValidationFailureException)
Because Exception could be thrown for any other reason